README.md

```diff
@@ -59,12 +59,10 @@ Edit your `config.yml` file and add the necessary configuration parameters as
 follows:
 
 ```yaml
-# You can get the consumer key and token and the respective tokens as described on https://developer-docs.bynder.com/API/Authorization-and-authentication/#create-a-consumer
```

Comment:
Can we have an `Upgrade from 1.x to 2.x` section here. I know it only says that the only thing you need to do is replace the config from the 4 values to our new permanent token solution but still, I consider this valuable for people upgrading.